src/style.css

```
@@ -0,0 +1,116 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img-box {
+  margin: 32px 19px 40px;
+  display: flex;
+  width: 160px;
+  height: 134px;
+}
+
+.product-img {
+  margin: 0 auto;
+  width: 160px;
+  height: 100%;
```

add width/height only on image itself, not on its container

```suggestion
  width: 160px;
  height: 134px;
```